<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <node pkg="snap_cam" type="snap_cam_publisher_node" name="snap_cam_optflow_publisher" output="screen">
    <!--
    0: optical flow camera_type
    1: highres camera_type
    -->
    <param name="camera_type" value="0"/>
    <!--
    optical flow: 0: QVGASize (320,240)
                  1: VGASize (640,480)
    -->
    <param name="camera_resolution" value="0"/>
    <!--
    optical flow: 0: 15 1: 24 2: 30 3: 60 4: 90 5: 120
    -->
    <param name="camera_fps" value="0"/>
    <param name="topic_name" value="snap_cam/optical_flow/image"/>
  </node>
</launch>
